jax/_src/api.py

```diff
@@ -147,8 +148,40 @@ def _update_debug_special_thread_local(_):
 float0 = dtypes.float0
 
 
+def _allow_deprecated_jit_signature(f: F) -> F:
```

Comment:
nit: Add a comment saying when to delete this by? (July 15, 2025)

Reply:
Done